### ISSUE TITLE
Auto-Updater

### ISSUE THREAD

**Issue by kostia** [kostia/VexEd#8]
See: https://github.com/jenslind/electron-gh-releases.

**Comment by kostia** [comment]
Blocked by https://github.com/kostia/VexEd/issues/7#issuecomment-155833266.

**Comment by kostia** [comment]
Really complicated... Postpone to 1.1.0.
